Trustroots/trustroots
# Internationalization (i18n) with react-i18next

#### Proposed Changes

* Testing an react-i18next library. So far works fine.

Largely [following](https://react.i18next.com/guides/quick-start) [tutorials](https://react.i18next.com/guides/step-by-step-guide) on https://react.i18next.com.

#### Testing Instructions

* `git checkout react-i18next`
* `npm install`
* `npm start`
* go to http://localhost:3000/volunteering
* try to switch language from `eng` to `cze` and observe the magic

Fixes #492

TODO:
- [x] hide behind a feature flag
- [ ] test translation extraction tools and translating tools
